src/express/commands/gov-tests.js

```diff
@@ -0,0 +1,496 @@
+import { loadDevnetConfig } from '../common/config-utils.js'
+import { timer } from '../common/time-utils.js'
+import {
+  runSshCommand,
+  runSshCommandWithReturn,
+  runSshCommandWithoutExit,
+  maxRetries
+} from '../common/remote-worker.js'
+import dotenv from 'dotenv'
+
+export async function sendGovTestsCommand() {
+  dotenv.config({ path: `${process.cwd()}/.env` })
+  const devnetType =
+    process.env.TF_VAR_DOCKERIZED === 'yes' ? 'docker' : 'remote'
+
+  const doc = await loadDevnetConfig(devnetType)
+  let machine0
+
+  if (doc.numOfBorValidators > 0) {
+    machine0 = doc.devnetBorHosts[0]
+    console.log('📍Monitoring the first node', doc.devnetBorHosts[0])
+  } else if (devnetType === 'remote') {
+    machine0 = doc.devnetErigonHosts[0]
+    console.log('📍Monitoring the first node', doc.devnetErigonHosts[0])
+  } else {
+    console.log('📍No nodes to monitor, please check your configs! Exiting...')
+    process.exit(1)
+  }
+
+  for (const machine of doc.devnetBorHosts) {
+    console.log(`📍Processing host: ${machine}`)
+    try {
+      // Fetch base64-encoded private key
```

Comment:
This key fetching and encoding/decoding is repeated in more than 1 test. Maybe have a utility? Same for keyring imports